src/cluster.bootstrap/Akka.Management.Cluster.Bootstrap/AkkaHostingExtensions.cs

```diff
@@ -123,29 +123,14 @@ public static class AkkaHostingExtensions
             if (autoStart)
             {
                 // Inject ClusterBootstrapProvider into akka.extensions
-                if (builder.Configuration.IsEmpty)
-                {
-                    var config = (Config)$"akka.extensions=[\"{typeof(ClusterBootstrapProvider).AssemblyQualifiedName}\"]";
-                    builder.AddHocon(config, HoconAddMode.Prepend);
-                }
-                else
-                {
-                    var extensions = builder.Configuration.Value.GetStringList("akka.extensions").ToList();
-                    if (extensions.All(s => !s.Contains(nameof(ClusterBootstrapProvider))))
-                    {
-                        extensions.Add(typeof(ClusterBootstrapProvider).AssemblyQualifiedName);
-                        var config = (Config)$"akka.extensions=[{string.Join(",", extensions.Select(s => $"\"{s}\""))}]";
-                        builder.AddHocon(config, HoconAddMode.Prepend);
-                    }
-                }
-            }
-            else
-            {
-                builder.AddHocon(
-                    "akka.management.http.routes.cluster-bootstrap = \"Akka.Management.Cluster.Bootstrap.ClusterBootstrapProvider, Akka.Management.Cluster.Bootstrap\"", 
-                    HoconAddMode.Prepend);
+                builder.WithExtensions(typeof(ClusterBootstrapProvider));
```

Comment:
nice - do we need to release a new version of Akka.Hosting so you can take advantage of the strongly typed version of this or does that not matter?